.github/workflows/gh-pages-scraping.yml

```diff
@@ -47,3 +47,27 @@ jobs:
           -e MEILISEARCH_API_KEY=$API_KEY \
           -v $CONFIG_FILE_PATH:/docs-scraper/config.json \
           getmeili/docs-scraper:v0.9.0 pipenv run ./docs_scraper config.json
+
+  scrap-github:
+    needs: scrap-docs
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Connect to GitHub docker registry
+      uses: azure/docker-login@v1
+      with:
+        login-server: docker.pkg.github.com
+        username: ${{ secrets.GITHUB_DOCKER_REGISTRY_USERNAME }}
+        password: ${{ secrets.GITHUB_DOCKER_REGISTRY_PASSWORD }}
+    - name: Run github-issues-scraper
+      env:
+        HOST_URL: ${{ secrets.MEILISEARCH_HOST_URL }}
+        API_KEY: ${{ secrets.MEILISEARCH_API_KEY }}
+      run: |
+        docker run -t --rm \
+          -e GH_LOGIN=$GITHUB_LOGIN \
+          -e GH_TOKEN=$GITHUB_TOKEN \
+          -e GH_REPOSITORY=$  HOST_URL \
+          -e MEILISEARCH_URL=$HOST_URL \
+          -e MEILISEARCH_API_KEY=$API_KEY \
+          docker.pkg.github.com/meilisearch/github-issues-scraper/app:latest
```

Comment:
I think we should avoid using `latest` in production 🙂
I saw there is no release on `github-issues-scraper`, is it complicated to add one and add a tag to the docker image?